tracelogging/include/g3log_tracelogging.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+#include <g3log/logmessage.hpp>
+
+// Trace Logging
+#include <TraceLoggingProvider.h>
+
+TRACELOGGING_DECLARE_PROVIDER(traceProvider);
+
+#define TraceLogWrite(eventName, ...) \
+    _TlgWrite_imp(_TlgWrite, \
+    traceProvider, eventName, \
+    (NULL, NULL), \
+    __VA_ARGS__)
+
+// *** BEGIN EXAMPLE CLIENT TRACELOGGING DEFINITION ***
```

Comment:
For readability I would suggest keeping the example within 
```
/** 

*/
```

And removing the `//`

Reply:
Another good suggestions.  I made the change.